config/global-constants/src/lib.rs

```diff
@@ -11,7 +11,6 @@
 /// Definitions of global cryptographic keys (e.g., as held in secure storage)
 pub const APTOS_ROOT_KEY: &str = "aptos_root";
 pub const CONSENSUS_KEY: &str = "consensus";
-pub const EXECUTION_KEY: &str = "execution";
```

Comment:
What was this and why was it removed?

Reply:
it's some legacy stuff that's not used anymore